<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoWidget</class>
 <widget class="QWidget" name="videoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>122</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>696</width>
    <height>122</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>696</width>
    <height>122</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>104</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>92</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>122</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>122</height>
      </size>
     </property>
     <widget class="QLabel" name="currenTime">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QSlider" name="videoSlider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>500</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal{
height: 4px;
background-color:rgba(66,80,96,1) ;
border: 1px solid #566576;
border-radius: 0px;
padding-left:-1px;
padding right:-1px;}
        
QSlider::sub-page:horizontal{
height: 4px;
background: transparent;
border: 1px solid transparent;
border-radius: 2px;
}
  
QSlider::handle:horizontal
{
    width: 22px;
    background-color:rgba(255,255,255,1);
    margin-top: -10px;
    margin-bottom: -10px;
border-radius: 5px;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="adjustWidget" native="true">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>69</y>
        <width>248</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>248</width>
        <height>4</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>248</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>64</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="CArmLabel" name="retreat">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CArmLabel" name="playAndDisplay">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="CArmLabel" name="advance">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CArmLabel</class>
   <extends>QLabel</extends>
   <header location="global">CArmControl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
